src/app/account-settings/components/settings-security-alternate-sign-in/settings-security-alternate-sign-in.component.html

```diff
@@ -13,49 +13,79 @@
   >
     Learn more about using alternate accounts to sign in to ORCID
   </a>
-  <table
-    *ngIf="accounts.length"
-    mat-table
-    [dataSource]="accounts"
-    class="mat-elevation-z0"
-  >
-    <ng-container matColumnDef="provider">
-      <th mat-header-cell *matHeaderCellDef i18n="@@account.account">
-        Account
-      </th>
-      <td mat-cell *matCellDef="let element">
-        {{ element?.idpName }}
-      </td>
-    </ng-container>
+  <ng-container *ngIf="accounts.length && this.isMobile">
```

Comment:
Adding a separate display for mobile.
Just because tables are ready bad on mobile